lib/event.rb

```diff
@@ -0,0 +1,44 @@
+require 'pry'
+
+class Event
+  attr_reader :name,
+              :ages
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    ages.max
+  end
+
+  def min_age
+    ages.min
+  end
+
+  def average_age
+    ages.sum.to_f / ages.count
+  end
+
+  def standard_deviation_age
```

Comment:
This method is doing ALOT - too much responsibility for one method!  Break this into smaller methods!